snippets/operator-group-unique-name.adoc

```diff
@@ -0,0 +1,18 @@
+// Text snippet included in the following assemblies:
+//
+// * list of assemblies where this text snippet is included
+//
+// Text snippet included in the following modules:
+//
+// * list of modules where this text snippet is included
+
+:_content-type: SNIPPET
+
+[WARNING]
+====
+You must use a unique name that does not conflict with any of the existing cluster role bindings on a cluster or other Operator groups on the cluster.
+
+If your Operator group shares a name with an existing cluster role binding, Operator Lifecycle Manager (OLM) overwrites the previously existing cluster role. This can lead to unpredictable behaviour, such as getting locked out of the cluster.
```

Comment:
The key thing is if there are any existing CRB that have the name of `<operator-group>-view` or `<operator-group>-edit` or `<operator-group>-admin`.

If the OG is named `cluster-admin`, it's actually _ok_ because the CRBs are then named e.g. `cluster-admin-admin`, which shouldn't conflict.